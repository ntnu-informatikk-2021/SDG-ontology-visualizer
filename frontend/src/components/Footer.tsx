import React from 'react';
import { Flex } from '@chakra-ui/react';

const Footer = () => (
  <Flex
    className="footer"
    bgGradient="linear(to-r, green.200, pink.500)"
    w="100%"
    p={4}
    color="white"
  >
    <p>Footer content</p>
  </Flex>
);

export default Footer;
