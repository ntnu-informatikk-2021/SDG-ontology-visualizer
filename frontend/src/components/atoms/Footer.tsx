import { Center, Heading } from '@chakra-ui/react';
import React from 'react';
import { useSelector } from 'react-redux';
import { RootState } from '../../state/store';

const Footer = () => {
  const { isFullscreen } = useSelector((state: RootState) => state.fullscreenStatus);

  if (isFullscreen) return <></>;

  return (
    <footer>
      <Center bg="cyan.700" w="100%" p={4} color="white" h="15vh" mt="20">
        <Heading size="md">TRDK03-UN</Heading>
      </Center>
    </footer>
  );
};

export default Footer;
