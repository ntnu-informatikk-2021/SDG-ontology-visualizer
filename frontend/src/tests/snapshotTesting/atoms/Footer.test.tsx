import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../../../components/atoms/Footer';

it('renders when there are no items', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
