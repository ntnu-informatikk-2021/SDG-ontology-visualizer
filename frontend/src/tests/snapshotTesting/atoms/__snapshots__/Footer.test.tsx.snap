// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there are no items 1`] = `
<footer>
  <div
    className="css-zs5lze"
  >
    <h2
      className="chakra-heading css-0"
    >
      TRDK03-UN
    </h2>
  </div>
</footer>
`;
